backend/components/section/contacts-section.json

```diff
@@ -0,0 +1,26 @@
+{
+  "collectionName": "components_section_contacts_sections",
+  "info": {
+    "name": "contactsSection",
+    "icon": "address-book",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "title": {
+      "type": "string"
+    },
+    "subtitle": {
+      "type": "string"
+    },
+    "email": {
+      "type": "string"
+    },
+    "sectionTitle": {
+      "type": "string"
+    },
+    "areBubblesActive": {
+      "type": "boolean"
+    }
```

Comment:
Do we need bubbles for contacts?

Reply:
Yes!
![image](https://user-images.githubusercontent.com/58849430/128313296-70de8a84-38e2-4ceb-8862-fd243f7d5f03.png)

Reply:
@BaggioGiacomo Oh gotcha, you meant those bubbles, the my question would be, do we need to have a setting for these? I don't think this will be reused somewhere else without bubbles

Reply:
Nope, i've just remove this feature.